src/api/timer/TimerController.jsx

```diff
@@ -1,18 +1,10 @@
 import timerService from '../../services/TimerService';
 
-/**
- * @class TimerController
```

Comment:
![image](https://github.com/Kyjor/banFlow/assets/13784123/08b1f22f-efde-45bc-af8a-e0cc79db1169)
 
These give hover over information.

Reply:
This is a good example of why I want to remove these high level comments for now. That controller doesn't actually create anything. It's a wrapper around the `TimerService`, which itself is just a wrapper around `LokiService`. And more things use `LokiService` directly than going through the `TimerController` here.

Reply:
That's why I said they should be `updated, not removed`.  They were initially created as placeholders. The only use of `LokiService` directly is `initialization` on the timer page, main page, and project page. Every other call (data access and writing) is through controllers and  services.  I'm fine with the removal of the controller/service explanations for now, but they should be added back in the near future. The function descriptions should all be readded now though.